app/views/pages/dashboard.html.haml

```diff
@@ -13,6 +13,7 @@
             .row
               %h4.col-md-4.col-md-5-offset
                 = link_to shop.name, shop_path(shop)
+                %strong &euro; #{shop.balance/ 100.00}
```

Comment:
Let's have a model method that would return this rather than always calculating this on the view everywhere?